pkgcfg/package-configuration/apache/httpd.conf

```diff
@@ -0,0 +1,63 @@
+ServerAdmin             "root@localhost"
+ServerName              "devbox-apache"
+Listen                  "${HTTPD_PORT}"
+PidFile                 "${HTTPD_CONFDIR}/apache.pid"
+
+LoadModule mpm_event_module modules/mod_mpm_event.so
+LoadModule authz_host_module modules/mod_authz_host.so
+LoadModule authz_core_module modules/mod_authz_core.so
+LoadModule auth_basic_module modules/mod_auth_basic.so
+LoadModule mime_module modules/mod_mime.so
+LoadModule headers_module modules/mod_headers.so
+LoadModule unixd_module modules/mod_unixd.so
+LoadModule status_module modules/mod_status.so
+LoadModule proxy_module modules/mod_proxy.so
+LoadModule proxy_fcgi_module modules/mod_proxy_fcgi.so
+LoadModule dir_module modules/mod_dir.so
+LoadModule alias_module modules/mod_alias.so
+
+<IfModule unixd_module>
+    User daemon
+    Group daemon
+</IfModule>
+
+<Directory />
+    AllowOverride none
+    Require all denied
+</Directory>
```

Comment:
what does this do?

Reply:
it defaults all directories to access denied (and later whitelists just `web`)